crates/typst-library/src/compute/data.rs

```diff
@@ -130,11 +130,19 @@ pub fn csv(
     #[named]
     #[default]
     delimiter: Delimiter,
+    /// Whether to use the header row in the CSV file to generate labels the fields in the columns.
+    /// When using this option, each row in the CSV file will be represented as an dictionary, where the header field is the key and the column field is the value.
```

Comment:
Also grammar:

```suggestion
    /// When using this option, each row in the CSV file will be represented as a dictionary, where the header field is the key and the column field is the value.
```

In addition, the following line says _header rows_. Should it be _the header row_, because there is only one header row?

[Doc of `builder.has_headers`](https://docs.rs/csv/latest/csv/struct.ReaderBuilder.html#method.has_headers):

> Whether to treat _the first row_ as a special header row.